src/linkAttributes/linkAttributes.ts

```diff
@@ -105,6 +105,7 @@ function updateLinkExtraAttributes(app: App, settings: SuperchargedLinksSettings
 }
 
 export function updateDivExtraAttributes(app: App, settings: SuperchargedLinksSettings, link: HTMLElement, destName: string, linkName?: string) {
+    if (link.parentElement.getAttribute("class").contains('mod-collapsible')) return; // Bookmarks Folder
```

Comment:
This line just ignores `mod-collapsible` elements in the bookmarks-pane (which amounts to the "folders" you can create there)